articleList.scss

```diff
@@ -0,0 +1,36 @@
+@import "../variables";
```

Comment:
@import is okey for now, but according Sass docs, @import will be removed, so its better to use @use rule